docs/GE_Checklist.md

```diff
@@ -149,8 +149,8 @@ for new and incubated enablers listed above **MUST** be met.
 
 -   A Stack Overflow tag for the component is registered (Ask FF Staff)
 -   Where it is suitable to provide a global instance of a service, a public
-    instance is deployed in the FIWARE Lab and maintained by the FF team (Ask FF
-    Staff)
+    instance may be deployed in the FIWARE Lab and maintained by the FF team
```

Comment:
MAY? (capitalized)

Reply:
NTC - there is a reason I didn't captialize the MAY there in the text - it isn't a MAY requirement. its more "if this makes sense to do then do it" - it **may** or **may not** be necessary, but you won't lose points if you don't